tests/themes-project/themes/blog_theme/theme.ini

```diff
@@ -0,0 +1,15 @@
+name = "Blog theme"
+license = "MIT"
+licenselink = ""
+description = "And Example blog theme for testing"
+homepage = ""
+tags = ["simple", "minimal", "blog"]
+features = ["blog"]
+lektor_required_version = 3.1
+
+[author]
```

Comment:
This indentation style doesn't follow the style that Lektor currently has for `.lektorproject` and other `.ini` files as far as I can see.